mods/content/sexcon/datum/sexcon_helpers.dm

```diff
@@ -0,0 +1,83 @@
+// SCREEN OVERLAYS //
+
+/obj/screen/fullscreen/love
+	icon_state = "lovehud"
+	layer = CRIT_LAYER
+	alpha = 0
+
+/obj/screen/fullscreen/love/Initialize(mapload, mob/_owner, ui_style, ui_color, ui_alpha)
+	. = ..()
+	animate(src, alpha = 255, time = 30)
+
+// MOB HELPERS //
+
+/mob/living/human
+	var/can_do_sex = TRUE
+	var/virginity = FALSE
+	/// Whether the mob is actively conducting a sex act.
+	var/doing = FALSE
+	var/datum/sex_controller/sexcon
+
+/mob/living/human/Initialize(mapload, species_uid, datum/mob_snapshot/supplied_appearance)
+	. = ..()
+	if(get_config_value(/decl/config/toggle/sexcon))
+		sexcon = new(src)
+
+/mob/living/human/Destroy()
+	. = ..()
+	QDEL_NULL(sexcon)
+
+/mob/living/human/verb/start_sexcon()
```

Comment:
i recommend pulling the insides of this proc into a proc like `start_sexcon(mob/living/human/target)` and renaming this to like, `start_sexcon_verb()`. then you can do something like
```dm
/mob/living/human/handle_mouse_drop(atom/over, mob/user)
	. = ..()
	if(!. && ishuman(over) && ishuman(user) && user == src && user != over)
		start_sexcon(over)
		return TRUE
```
to give the middle-mouse-drag interaction that other codebases have